virtru-components/sjcl
# WS-2485 update crypto lib

I've left the file unminified, since all our other tool will minify it.

I ran `./configure  --without-all --with-cbc --with-codecBytes --with-random` to generate the file with the barebones that we use.
- changes: _minor_
  - WS-2485
    - _minor_: Updated to latest SJCL and generating a new file 

To test:
IE10 doesn't have native crypto, so SJCL should be used. All other places will use native crypto in secure-lib. The other way to test is to not use the native crypto by manually forcing it during debugging or to use secure-lib pre-native.
